workloads/network-perf/run_multus_network_tests_fromgit.sh

```diff
@@ -26,7 +26,8 @@ if [[ ${PAIR} ]]; then
 fi
 
 if ${MULTUS} ; then
-oc -n my-ripsaw delete benchmark/uperf-benchmark
+oc -n my-ripsaw delete benchmark/uperf-benchmark-multus-network --wait
+fi
```

Comment:
This fi shouldn't be here. the if matches to the fi from line 123

Reply:
It also begs the question as to why we have the if there to being w/ given this is the multus test. You wouldn't be running it if you didn't want it. Thoughts on getting rid of the surrounding if?

Reply:
Makes sense @dry923, removed it.